ChangeEveryLetterToTheNextLetter.rb

```diff
@@ -0,0 +1,47 @@
+=begin 
+Write a function that changes every letter to the next letter:
+
+"a" becomes "b"
+"b" becomes "c"
+"d" becomes "e"
+and so on ...
+
+Examples
+
+move("hello") ➞ "ifmmp"
+
+move("bye") ➞ "czf"
+
+move("welcome") ➞ "xfmdpnf"
+Notes
+There will be no z's in the tests.
+Algorithm:
+1. Given a word, Lets take the first alphabet and find the next alphabet to it according to alphabetical order. 
+2. similarly do it for the rest of the letter. 
+3. merge all the found alphabets as one word.
+=end 
+
+
+
+
+def move(word)
+  arr = []
+  word = word.downcase.scan /\w/ 
+  alphabet = Hash.new
+  ('a'..'z').each_with_index do |key, value| 
+    alphabet[key] = value 
```

Comment:
hash is a good idea. you get O(1) look up. so great for performance. but why are you storing the index value in the hash? 

you code below gets the index of it and then finds the next element with that but it need to loop through the hash to find the key. You lose the benefit of using the hash. 

How about you have a hash that look like this? 

```
{
  'a': 'b',
  'b': 'c',
  'c': 'd',
   ...
}
```

So when you have the key, you know immediately what that should be translated into.